src/qinterface/protected.cpp

```diff
@@ -243,4 +243,12 @@ bool QInterface::IsIdentity(const complex* mtrx)
     return true;
 }
 
+#if ENABLE_UINT128
+std::ostream& operator<<(std::ostream& left, const __uint128_t& right)
+{
+    left << (const uint64_t&)right;
```

Comment:
Do you really want to truncate the value like this?

Reply:
I asked myself the same thing, but the other option is slicing it up to manually print it as decimal. I'll consider doing exactly that, but it might become practically relevant only when I finish something else I'm working on now, in which case it could go in with that PR.

Reply:
See #251